artemis/src/main/java/com/artemis/EntityManager.java

```diff
@@ -91,35 +103,25 @@ void updateCompositionIdentity(EntityEdit edit) {
 	int compositionIdentity(BitSet componentBits) {
 		int identity = identityResolver.getIdentity(componentBits);
 		if (identity > highestSeenIdentity) {
-			subscriptionManager.processComponentIdentity(identity, componentBits);
+			world.getSystem(AspectSubscriptionManager.class)
```

Comment:
We do this everywhere, deserves its own getter on world. We pre-resolve it into a field anyway.